kivymd/uix/dialog.py

```diff
@@ -181,7 +181,11 @@ def show_alert_dialog(self):
 
 
 class BaseDialog(ThemableBehavior, ModalView):
-    radius = ListProperty([7, 7, 7, 7])
+    radius = ListProperty(
+        [
+            dp(7),
+        ]
+    )
```

Comment:
This is an error - the radius is intended to change the value of all angles, not just one. The four-element list means that you can set different values for any angles.

Reply:
https://github.com/kivymd/KivyMD/blob/93d6823b3eb6cbd6574f81852536cc548846acad/kivymd/uix/menu.py#L905
https://github.com/kivymd/KivyMD/blob/93d6823b3eb6cbd6574f81852536cc548846acad/kivymd/uix/tooltip.py#L172

before this [commit](https://github.com/kivymd/KivyMD/commit/69680954753c9f2e7cc57504bb918940ea9af5a8#diff-d808ef41d27ee485ae6d3f04e2cb96c5d3774c2696ffbc3299d33b15e7cd3ab3) in mobile the edges are sharp
after the edges are looks same in all platforms